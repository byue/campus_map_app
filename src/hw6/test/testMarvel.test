# Tests various inputs for the marvel graph. Expected outputs were found by
# browsing the marvel.tsv data.

LoadGraph marvel marvel.tsv

FindPath marvel HOFFMAN CAPTAIN_AMERICA

FindPath marvel SENTRY_IV/VAL BLURRY

FindPath marvel BLURRY SENTRY_IV/VAL

FindPath marvel W'KABI LYNNE,_MONICA

FindPath marvel CAPTAIN_AMERICA HAWK

FindPath marvel HAWK HAWK

FindPath marvel HAWK @#$