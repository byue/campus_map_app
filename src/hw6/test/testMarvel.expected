# Tests various inputs for the marvel graph. Expected outputs were found by
# browsing the marvel.tsv data.

loaded graph marvel

path from HOFFMAN to CAPTAIN AMERICA:
no path found

path from SENTRY IV/VAL to BLURRY:
SENTRY IV/VAL to ROM, SPACEKNIGHT via SK 1
ROM, SPACEKNIGHT to ANGEL/WARREN KENNETH via COC 1
ANGEL/WARREN KENNETH to BANSHEE/SEAN CASSIDY via CX 1
BANSHEE/SEAN CASSIDY to BLURRY via GENX 40

path from BLURRY to SENTRY IV/VAL:
BLURRY to ALBERT via GENX 42
ALBERT to BEAST/HENRY &HANK& P via W2 53
BEAST/HENRY &HANK& P to ROM, SPACEKNIGHT via COC 1
ROM, SPACEKNIGHT to SENTRY IV/VAL via SK 1

path from W'KABI to LYNNE, MONICA:
W'KABI to LYNNE, MONICA via BP3 15

path from CAPTAIN AMERICA to HAWK:
CAPTAIN AMERICA to HAWK via A '00

path from HAWK to HAWK:

unknown character @#$